.github/workflows/tests.yml

```diff
@@ -424,32 +424,32 @@ jobs:
   #       pytest ${{ env.PYTEST_PARAMS }} -m "not integration" ${{ env.SUITES_EXCLUDED_FROM_WINDOWS }} test/document_stores/ --document_store_type=weaviate
 
 
-  pinecone-tests-linux:
-    needs:
-     - mypy
-     - pylint
-    runs-on: ubuntu-latest
-    if: contains(github.event.pull_request.labels.*.name, 'topic:pinecone') || !github.event.pull_request.draft
+  # pinecone-tests-linux:
```

Comment:
Nit: can you add an extra `FIXME` line explaining this is temporary and will be reverted soon?